go.mod

```diff
@@ -169,6 +161,7 @@ require (
 	github.com/apache/thrift v0.21.0 // indirect
 	github.com/ardielle/ardielle-go v1.5.2 // indirect
 	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5 // indirect
+	github.com/aws/aws-sdk-go v1.55.7 // indirect
```

Comment:
Please add a depguard rule in `.golangci.yml` to deny usage of `github.com/aws/aws-sdk-go`